hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestMetaRegionReplicaReplication.java

```diff
@@ -477,7 +495,7 @@ public void testHBaseMetaReplicaGets() throws Exception {
 
       // There are read requests increase for primary meta replica.
       // For rest of meta replicas, there is no change as regionMove will tell the new location
-      primaryIncreaseReplicaNoChange(readReqsForMetaReplicasAfterGet,
+      primaryIncreaseReplicaNoChange(readReqsForMetaReplicasAfterGetAllLocations,
```

Comment:
Mind explaining this UT a bit? I can not get the point here why there is no change, since we have moved a region?